test/python/qobj/test_pulse_converter.py

```diff
@@ -152,7 +152,7 @@ def test_set_frequency(self):
             name='setf',
             ch='d0',
             t0=0,
-            frequency=8.0
+            frequency=8e-9
```

Comment:
```suggestion
            frequency=8.
```
and line 149 should become `instruction = SetFrequency(8.0e9, DriveChannel(0))`

Reply:
sorry, I missed it. Fixed in https://github.com/Qiskit/qiskit-terra/pull/4399/commits/33fe46d2fd645702fe5c412e01865aa3e852e46f.